docs/source/en/api/loaders/lora.md

```diff
@@ -79,4 +79,8 @@ To learn more about how to load LoRA weights, see the [LoRA](../../using-diffuse
 
 ## LoraBaseMixin
 
-[[autodoc]] loaders.lora_base.LoraBaseMixin
\ No newline at end of file
+[[autodoc]] loaders.lora_base.LoraBaseMixin
+
+## WanLoraLoaderMixin
+
+[[autodoc]] loaders.lora_pipeline.WanLoraLoaderMixin
```

Comment:
Could consider extending this to include LoRA loaders too:
https://github.com/huggingface/diffusers/pull/7763/f